ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh

```diff
@@ -134,7 +134,17 @@ fi
 arch_instance_type=$(echo -n "${CONTROL_PLANE_INSTANCE_TYPE}" | cut -d . -f 1)
 BOOTSTRAP_NODE_TYPE=${arch_instance_type}.large
 
-workers=3
+if [[ -n "${COMPUTE_NODE_REPLICAS}" ]]; then
+  if [[ "${COMPUTE_NODE_REPLICAS}" -le 0 ]]; then
+    workers=0
+  elif [[ "${COMPUTE_NODE_REPLICAS}" -gt 3 ]]; then
+    workers=3
+  else
```

Comment:
It's possible we should return an error here that prevents the job from executing rather than clamping to a floor value of 3. I'll leave this up to the aws platform reviewer though. It doesn't matter much to me one way or the other for our purposes.